spotbugs/src/main/java/edu/umd/cs/findbugs/BugInstance.java

```diff
@@ -1617,11 +1608,7 @@ public BugInstance addSourceLine(SourceLineAnnotation sourceLine) {
      */
     @Nonnull
     public BugInstance addSourceLine(BytecodeScanningDetector visitor, int pc) {
-        SourceLineAnnotation sourceLineAnnotation = SourceLineAnnotation.fromVisitedInstruction(visitor.getClassContext(),
-                visitor, pc);
-        if (sourceLineAnnotation != null) {
-            add(sourceLineAnnotation);
-        }
+        add(SourceLineAnnotation.fromVisitedInstruction(visitor.getClassContext(), visitor, pc));
```

Comment:
All of these cases, the variable can never be null, so the null check is not necessary.